src/mirror.envd

```diff
@@ -0,0 +1,159 @@
+# pypi source
+# see https://mirrors.cernet.edu.cn/list/pypi-packages
+PYPI_URL = {
+    # North China
+    "tuna": "https://pypi.tuna.tsinghua.edu.cn/simple",
+    "pku": "https://mirrors.pku.edu.cn/pypi/simple",
+    "bfsu": "https://mirrors.bfsu.edu.cn/pypi/web/simple",
+    "nyi": "https://mirror.nyist.edu.cn/pypi/web/simple",
+    # Northeast China
+    "dnui": "https://mirrors.neusoft.edu.cn/pypi/web/simple",
+    # Southwest China
+    "lzu": "https://mirror.lzu.edu.cn/pypi/web/simple",
+    "xjtu": "https://mirrors.xjtu.edu.cn/pypi/simple",
+    # East China
+    "sjtu": "https://mirror.sjtu.edu.cn/pypi/web/simple",
+    "nju": "https://mirror.nju.edu.cn/pypi/web/simple",
+    "njt": "https://mirrors.njtech.edu.cn/pypi/web/simple",
+    # South China
+    "sus": "https://mirrors.sustech.edu.cn/pypi/simple",
+}
+
+# anaconda source
+# see https://mirrors.cernet.edu.cn/list/conda
+CONDA_URL = {
+    # North China
+    "tuna": "https://mirrors.tuna.tsinghua.edu.cn/anaconda",
+    "bfsu": "https://mirrors.bfsu.edu.cn/anaconda",
+    # Southwest China
+    "lzu": "https://mirror.lzu.edu.cn/anaconda",
+    # East China
+    "zju": "https://mirrors.zju.edu.cn/anaconda",
+    "nju": "https://mirror.nju.edu.cn/anaconda",
+    "njt": "https://mirrors.njtech.edu.cn/anaconda",
+    "njup": "https://mirrors.njupt.edu.cn/anaconda",
+}
+
+# ubuntu apt source
+# see also https://mirrors.cernet.edu.cn/list/ubuntu
+APT_URL = {
+    # North China
+    "tuna": "https://mirrors.tuna.tsinghua.edu.cn/ubuntu/",
+    "pku": "https://mirrors.pku.edu.cn/ubuntu",
+    "bfsu": "https://mirrors.bfsu.edu.cn/ubuntu",
+    "isc": "https://mirror.iscas.ac.cn/ubuntu",
+    "nyi": "https://mirror.nyist.edu.cn/ubuntu",
+    # Southwest China
+    "lzu": "https://mirror.lzu.edu.cn/ubuntu",
+    "nwa": "https://mirrors.nwafu.edu.cn/ubuntu",
+    "xjtu": "https://mirrors.xjtu.edu.cn/ubuntu/",
+    # Northeast China
+    "dnui": "http://mirrors.neusoft.edu.cn/ubuntu",
+    # Southwest China
+    "cqu": "https://mirrors.cqu.edu.cn/ubuntu",
+    # East China
+    "ustc": "https://mirrors.ustc.edu.cn/ubuntu",
+    "sjtu": "https://mirror.sjtu.edu.cn/ubuntu",
+    "zju": "https://mirrors.zju.edu.cn/ubuntu",
+    "nju": "https://mirror.nju.edu.cn/ubuntu",
+    "njt": "https://mirrors.njtech.edu.cn/ubuntu",
+    "njup": "https://mirrors.njupt.edu.cn/ubuntu",
+    "qlu": "https://mirrors.qlu.edu.cn/ubuntu",
+    "sdu": "https://mirrors.sdu.edu.cn/ubuntu",
+    # South China
+    "sus": "https://mirrors.sustech.edu.cn/ubuntu",
+    "scau": "https://mirrors.scau.edu.cn/ubuntu",
+}
+
+UBUNTU_OS = {"ubuntu18.04": "bionic", "ubuntu20.04": "focal", "ubuntu22.04": "jammy"}
+
+CONDA_TEMPLATE = """
+channels:
+  - defaults
```

Comment:
I'm not sure if we can use the mirror of this channel. Refer to https://github.com/tensorchord/envd/pull/929#issue-1380523897.